src/Components/CollapsiblePanel/CollapsiblePanel.js

```diff
@@ -17,6 +22,10 @@ const propTypes = {
    */
   label: PropTypes.oneOfType([PropTypes.node, PropTypes.func])
     .isRequired,
+  /**
+   * Applies styling to indicate the panel content has error
+   */
+  isError: PropTypes.bool,
```

Comment:
I'm not sure this is the best approach since the panel isn't necessarily a form. I think just exposing a classname prop that the client can add to might be better.